src/swap/defi/0x/0xGasless.ts

```diff
@@ -175,10 +177,42 @@ export const make0xGaslessPlugin: EdgeCorePluginFactory = opts => {
             throw new Error(`Swap failed: ${apiSwapStatus.reason ?? 'unknown'}`)
           }
 
+          const assetAction: EdgeAssetAction = {
+            assetActionType: 'swap'
+          }
+          const orderId = apiSwapSubmition.tradeHash
+          const {
+            publicAddress: toWalletAddress
+          } = await swapRequest.toWallet.getReceiveAddress({
```

Comment:
This is really freaking me out. We never seem to request an address from the `toWallet` before this line, so how on earth are the funds arriving there? It *seems* like the gasless swaps are only within the same wallet, so do we need to add a `toWallet.id === fromWallet.id` check?